Testing/Data/SystemTest/PEARL/Calibration_Test/Calibration/pearl_absorp_sphere_10mm_newinst2_long.nxs.md5

```diff
@@ -0,0 +1 @@
+d1b66c634fc4e6046e8f75121b505518
```

Comment:
@FedeMPouzols  I have created a folder for `Calibration` test, would be a good idea creating a separate folder for `Focus` system test too?